src/object/srv_obj_remote.c

```diff
@@ -433,6 +438,12 @@ ds_obj_cpd_dispatch(struct dtx_leader_handle *dlh, void *arg, int idx,
 	if (remote_arg == NULL)
 		D_GOTO(out, rc = -DER_NOMEM);
 
+	sched_exec_time(&time2, NULL);
+	diff = time2 - time1;
+	if (unlikely(diff >= 2))
+		D_WARN("Alloc DRAM is too slow: %u msecs, sub_cnt %u, idx %u\n",
+		       diff, dlh->dlh_sub_cnt, idx);
```

Comment:
Looks 'diff >= 1' deserves a warning. Same to other warnings in this function.

Reply:
"diff >= 1" may cause more useless warning message. In DAOS-10131, we observed more long time watchdog. So if it was really caused by the IO request dispatching logic, then we will hit "diff >= 2" case.